### PR TITLE
Improves docs SEO

### PR DESCRIPTION
**Pull request by lucas-tortora**
# Description of change

* Added keywords, image and description meta tags to all docs
* Renamed and moved docs files for browser friendly descriptive urls according to menu structure
* Fixed title capitalization and hierarchy 
* Switched > (notes) for admonitions :::
* Added keywords, image and description for all docs.
* Made images clickable/expandable
* Made local copies of all images
* Alpha sorted Glossary

## Links to any relevant issues

* Resolves  iotaledger/docs.iota.org#39

## Type of change

- [x] Enhancement (a non-breaking change which adds functionality)
- [x] Documentation Fix

## How the change has been tested

Docs site was built and served locally using node 12.

## Change checklist

Add an `x` to the boxes that are relevant to your changes, and delete any items that are not.

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by lucas-tortora** [comment]
@iotaledger/tech-writers  could someone review this?

File: documentation/docs/apis/autopeering.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
+---
+description: The peering API allows retrieving basic information about autopeering using the /autopeering/neighbors endpoint or the GetAutopeeringNeighbors() function in the client lib.
+image: replaceWithGoShimmerLogo
```
Comment (aleksei-korolev) [inline documentation/docs/apis/autopeering.md:3]: we don't have the logo yet, right?

File: documentation/docs/implementation_design/packages_plugins.md [hunk 2]
```diff
@@ -1,4 +1,18 @@
...
+description: GoShimmer uses the adapter design pattern to easily switch between different implementations and internal interfaces just by using a different plugin, without having to rewrite the code using it.
+image: replaceWithGoShimmerLogo
+keywords:
+- dependency
+- plugins
+- plugin system
+- code
+- internal logic
+- package
+- adapter design pattern
```
Comment (aleksei-korolev) [inline documentation/docs/implementation_design/packages_plugins.md:11]: does it grab partial queries like "adapter"?

File: documentation/docs/tutorials/monitoring.md [hunk 3]
```diff
@@ -195,10 +217,11 @@ Note: you have to have root privileges with your user to carry out the following
 
 Prometheus now collects metrics from your node, but we need to setup Grafana to visualize the collected data.
 
-### Install and configure Grafana
+### Install and Configure Grafana
```
Comment (aleksei-korolev) [inline documentation/docs/tutorials/monitoring.md:220]:
```suggestion

### Install and Configure Grafana
```

**Review by aleksei-korolev — APPROVED** [review]

File: documentation/docs/implementation_design/packages_plugins.md (hunk 2 above)
Comment (lucas-tortora) [reply documentation/docs/implementation_design/packages_plugins.md:11]: It does, I'll add it .
```suggestion
- adapter design pattern
- adapter
```

File: documentation/docs/apis/autopeering.md (hunk 1 above)
Comment (lucas-tortora) [reply documentation/docs/apis/autopeering.md:3]: I've added the wiki icon just now, but let's wait and see if there's a btter

File: documentation/docs/faq.md [hunk 4]
```diff
@@ -12,7 +26,7 @@ Since non-conflicting transactions aren't even voted on, they materialize after
 
 You can access the global analysis dashboard in the [Pollen Analyzer](http://ressims.iota.cafe:28080/autopeering) which showcases the network graph and active ongoing votes on conflicts.
 
-## How Many Transactions Per Second(TPS) can GoShimmer Sustain?
+## How Many Transactions Per Second(TPS) Can GoShimmer Sustain?
```
Comment (apenzk) [inline documentation/docs/faq.md:29]:
```suggestion
## How Many Transactions per Second (TPS) Can GoShimmer Sustain?
```

Comment (apenzk) [reply documentation/docs/faq.md:29]: https://capitalizemytitle.com/

File: documentation/docs/protocol_specification/components/consensus_mechanism.md [hunk 5]
```diff
@@ -163,7 +188,8 @@ Finally, the green nodes issued **statement 3**, which is in `Branch 2`. Now the
 It is important to notice that the arrival order of the statements does not make a difference on the final outcome. Due to the fact that statements can be ordered objectively, every node in the network eventually comes to the same conclusion as to who is supporting which branch, even when nodes change their opinion.
 
 
-##### Calculation of Approval Weight
+##### Calculation of approval weight
```
Comment (apenzk) [inline documentation/docs/protocol_specification/components/consensus_mechanism.md:191]:
```suggestion
##### Calculation of Approval Weight
```

Comment (apenzk) [reply documentation/docs/protocol_specification/components/consensus_mechanism.md:191]: Approval Weight is a defined term, thus capitalized

File: documentation/docs/tutorials/manual_peering.md [hunk 6]
```diff
@@ -17,7 +30,7 @@ will keep reconnecting with known peers until it succeeds.
 In other words, the only thing that users have to do to be connected via manual peering is to 
 exchange their IP address with port and public key and set that information to known peers of their nodes and machines will do the rest.
 
-## How to Set Known Peers via Config File
+## How to Set Known Peers Via Config File
```
Comment (apenzk) [inline documentation/docs/tutorials/manual_peering.md:33]:
```suggestion
## How to Set Known Peers via Config File
```

Comment (apenzk) [reply documentation/docs/tutorials/manual_peering.md:33]: which style are you using?
Above "With" was capitalized which is e.g. APA
via is generally non-capitalized in most common styles.
https://capitalizemytitle.com/style/APA/

File: documentation/docs/apis/faucet.md [hunk 7]
```diff
@@ -1,3 +1,16 @@
+---
+description: TFaucet endpoint allows requesting funds from the Faucet.
```
Comment (lucas-tortora) [inline documentation/docs/apis/faucet.md:2]:
```suggestion 
The Faucet endpoint allows requesting funds from the Faucet.
```

File: documentation/docs/apis/ledgerstate.md [hunk 8]
```diff
@@ -1,4 +1,18 @@
+---
+description: The ledgerstate API provides endpoints to retrieve address details, unspent outputs for an address,get branch details , a list of child branches amongst others.
```
Comment (lucas-tortora) [inline documentation/docs/apis/ledgerstate.md:2]:
```suggestion
The ledgerstate API provides endpoints to retrieve address details, unspent outputs for an address, get branch details, and list of child branches amongst others.
```

File: documentation/docs/apis/ledgerstate.md [hunk 9]
```diff
@@ -1,4 +1,18 @@
+---
+The ledgerstate API provides endpoints to retrieve address details, unspent outputs for an address, get branch details, and list child branches amongst others.
```
Comment (lucas-tortora) [inline documentation/docs/apis/ledgerstate.md:2]:
```suggestion 
description: The ledgerstate API provides endpoints to retrieve address details, unspent outputs for an address, get branch details, and list child branches amongst others.
```

File: documentation/docs/welcome.md [hunk 10]
```diff
@@ -1,3 +1,12 @@
+---
+description: GoShimmer is a prototype node software exploring the implementation of IOTA 2.0. This wiki helps the reader to understand the basic concepts and ideas behind the Coordicide, and how its modules are implemented in GoShimmer.
```
Comment (luca-moser) [inline documentation/docs/welcome.md:2]:
```suggestion
description: GoShimmer is a prototype node software exploring the implementation of IOTA 2.0. This wiki helps the reader to understand the basic concepts and ideas behind Coordicide, and how its modules are implemented in GoShimmer.
```

File: documentation/docs/apis/faucet.md [hunk 11]
```diff
@@ -1,3 +1,16 @@
+---
+The Faucet endpoint allows requesting funds from the Faucet.
```
Comment (luca-moser) [inline documentation/docs/apis/faucet.md]:
```suggestion
description: The Faucet endpoint allows requesting funds from the Faucet.
```

**Review by luca-moser — CHANGES_REQUESTED** [review]

File: documentation/docs/apis/webAPI.md [hunk 12]
```diff
@@ -1,9 +1,20 @@
+---
+description: The web API interface allows access to functionality of the node software via exposed HTTPendpoints.
```
Comment (luca-moser) [inline documentation/docs/apis/webAPI.md:2]:
```suggestion
description: The web API interface allows access to functionality of the node software via exposed HTTP endpoints.
```

File: documentation/docs/tutorials/send_transaction.md [hunk 13]
```diff
@@ -1,16 +1,35 @@
-# How to send transaction
+---
+description: The simplest easiest way to create a transaction is to use ready solutions, such us GUI wallets. But you can also create transactions using the Go client library.
```
Comment (luca-moser) [inline documentation/docs/tutorials/send_transaction.md:2]:
```suggestion
description: The simplest easiest way to create a transaction is to use ready solutions, such as GUI wallets. But you can also create transactions using the Go client library. 
```

**Review by luca-moser — CHANGES_REQUESTED** [review]

**Review by luca-moser — APPROVED** [review]

**Pull request merged**